cadquery/cq_directive.py

```diff
@@ -183,7 +183,7 @@
 
     renderWindow.addRenderer(renderer);
     updateViewPort(container, renderer);
-    renderer.getActiveCamera().set({ position: [-1, -1, -0.7], viewUp: [0, 0, -1] });
+    renderer.getActiveCamera().set({ position: [-1, -1, -0.7], viewUp: [0, 0, 1] });
```

Comment:
The parametric enclosure example looks wrong now, in CQ-editor:
![screenshot2021-07-15-135316](https://user-images.githubusercontent.com/50230945/125728757-ef1474a1-fd30-4042-a71e-a1f84ce659f1.png)

In RTD: https://cadquery--822.org.readthedocs.build/en/822/examples.html#a-parametric-enclosure

Is something wrong further upstream that causes the Z axis to be flipped?

Reply:
I'm an idiot, I needed to select just the `result` object in CQ-editor. I thought the parametric enclosure was upside down but it just had the lid on it still.